### DISCUSSION TITLE
[Script request] The Lounge IRC

### DISCUSSION THREAD

**Discussion by devbyaccident** [community-scripts/ProxmoxVE#501] [Request script] [answered]
### Application Name

The Lounge

### Website

https://thelounge.chat/

### Description

The Lounge is an open-source web client for IRC chat. 

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by quantumryuu** [comment, marked as answer]
Assign me please! Already pushed a PR 😃

**Reply by devbyaccident** [reply]
Awesome, thanks @quantumryuu!

**Reply by quantumryuu** [reply]
You're welcome mate ❤️
